ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -18,7 +18,7 @@ trait JdbcConnectionProvider extends AutoCloseable {
     * is returned to the pool.
     * The block must not recursively call [[runSQL]], as this could result in a deadlock
     * waiting for a free connection from the same pool. */
-  def runSQL[T](block: Connection => T): T
+  def runSQL[T](description: String, block: Connection => T): T
```

Comment:
This spreads the logging concern across two classes, I'm not sure we should do this.

Reply:
Agreed this looks strange.

Reply:
why do we need it here as well?

Reply:
The original intent was for errors, i.e. [in the HikariJdbcConnectionProvider where errors are caught](https://github.com/digital-asset/daml/blob/master/ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/HikariJdbcConnectionProvider.scala#L87-L89) to log the sql description as well, not just regular sql trace [in the SqlExecutor](https://github.com/digital-asset/daml/blob/master/ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/util/SqlExecutor.scala#L43-L44). This helps in regular production in which trace is off, but errors are logged.